modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/UninitializedOperationMessageProcessor.java

```diff
@@ -0,0 +1,266 @@
+    operationMessageProcessorDelegate.doStop();
+  }
+
+  @Override
+  public void doDispose() {
+    operationMessageProcessorDelegate.doDispose();
+  }
+
+  @Override
+  public ExecutionMediator createExecutionMediator() {
+    return operationMessageProcessorDelegate.createExecutionMediator();
+  }
+
+  @Override
+  public ParameterValueResolver getParameterValueResolver() {
+    return operationMessageProcessorDelegate.getParameterValueResolver();
+  }
+
+  @Override
+  public void resolveParameters(CoreEvent.Builder eventBuilder,
+                                BiConsumer<Map<String, Supplier<Object>>, ExecutionContext> afterConfigurer)
+      throws MuleException {
+    operationMessageProcessorDelegate.resolveParameters(eventBuilder, afterConfigurer);
+  }
+
+  @Override
+  public void disposeResolvedParameters(ExecutionContext<OperationModel> executionContext) {
+    operationMessageProcessorDelegate.disposeResolvedParameters(executionContext);
+  }
+
+
+  @Override
+  protected Map<String, Object> getResolutionResult(CoreEvent event, Optional<ConfigurationInstance> configuration)
+      throws MuleException {
+
+    ValueResolvingContext valueResolvingContext = ValueResolvingContext.builder(event).build();
+    Object config = resolverSet.resolve(valueResolvingContext).asMap().get("config-ref");
+
+    Map resolvedParametersMap = resolvedParameters.entrySet().stream()
+        .filter(e -> e.getKey().equals("config-ref"))
+        .collect(Collectors.toMap(key -> key, value -> {
+          try {
+            return value.getValue().resolve(valueResolvingContext);
+          } catch (MuleException e) {
+            e.printStackTrace();
+          }
+          return null;
+        }));
+
+    return (Map) ImmutableMap.builder().putAll(resolvedParametersMap).put("config-ref", config).build();
+  }
+
+  @Override
+  public void setMuleContext(MuleContext context) {
+    operationMessageProcessorDelegate.setMuleContext(context);
+  }
+
+  @Override
+  public MetadataResult<MetadataKeysContainer> getMetadataKeys() throws MetadataResolvingException {
+    return operationMessageProcessorDelegate.getMetadataKeys();
+  }
+
+  @Override
+  public MetadataResult<ComponentMetadataDescriptor<OperationModel>> getMetadata() throws MetadataResolvingException {
+    return operationMessageProcessorDelegate.getMetadata();
+  }
+
+  @Override
+  public MetadataResult<ComponentMetadataDescriptor<OperationModel>> getMetadata(MetadataKey key)
+      throws MetadataResolvingException {
+    return operationMessageProcessorDelegate.getMetadata(key);
+  }
+
+  @Override
+  public Set<Value> getValues(String parameterName) throws ValueResolvingException {
+    return operationMessageProcessorDelegate.getValues(parameterName);
+  }
+
+  @Override
+  public ExtensionModel getExtensionModel() {
+    return operationMessageProcessorDelegate.getExtensionModel();
+  }
+
+  @Override
+  public List<ValueProviderModel> getModels(String providerName) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Don't override a deprecated method or explicitly mark it as "@Deprecated". ([squid:CallToDeprecatedMethod](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7719&issues=AWk7fTH-otIXnmsUoDr9&open=AWk7fTH-otIXnmsUoDr9)

[ISSUE_KEY]: AWk7fTH-otIXnmsUoDr9